tests/subworkflows/nf-core/annotation/ensemblvep/test.yml

```
@@ -1,7 +1,11 @@
 - name: ensemblvep annotation_ensemblvep
   command: nextflow run ./tests/subworkflows/nf-core/annotation_ensemblvep -entry annotation_ensemblvep -c ./tests/config/nextflow.config -c ./tests/subworkflows/nf-core/annotation_ensemblvep/nextflow.config
```

```suggestion
  command: nextflow run ./tests/subworkflows/nf-core/annotation_ensemblvep -entry annotation_ensemblvep -c ./tests/config/nextflow.config
```
These config paths can be removed, they weren't removed from modules to avoid changing too many files (see discussion in [#1562](https://github.com/nf-core/tools/issues/1562)). As we are starting fresh with subworkflows I think it's a good opportunity to avoid them :)

---

Good idea, done!